src/widgets/SelectWidget.js

```diff
@@ -90,6 +90,15 @@ const SelectWidget = (props) => {
   }
   const labels = uiSchema['ui:enumNames'] || schema.enumNames || values;
 
+  const onPickerConfirm = (val) => {
```

Comment:
'onPickerConfirm' was used before it was defined  
'onPickerSelect' was used before it was defined